src/sentry/sentry_metrics/indexer/postgres_v2.py

```diff
@@ -9,17 +9,25 @@
 from sentry.sentry_metrics.indexer.cache import indexer_cache
 from sentry.sentry_metrics.indexer.db import TABLE_MAPPING, IndexerTable
 from sentry.sentry_metrics.indexer.ratelimiters import writes_limiter
-from sentry.sentry_metrics.indexer.strings import REVERSE_SHARED_STRINGS, SHARED_STRINGS
+from sentry.sentry_metrics.indexer.static_strings import StaticStringIndexer as StaticStringIndexer
 from sentry.utils import metrics
 
 from .base import FetchType
 
+__all__ = "PostgresIndexer"
+
+
 _INDEXER_CACHE_METRIC = "sentry_metrics.indexer.memcache"
 _INDEXER_DB_METRIC = "sentry_metrics.indexer.postgres"
 # only used to compare to the older version of the PGIndexer
 _INDEXER_CACHE_FETCH_METRIC = "sentry_metrics.indexer.memcache.fetch"
 
 
+class PostgresIndexer(StaticStringIndexer):
+    def __init__(self) -> None:
+        super().__init__(PGStringIndexerV2())
```

Comment:
Why this ? Is it to avoid refactoring too many imports ?

Reply:
So that static string indexer doesn't have to live inside the postgres module and be more easily shared. Also I prefer to export the name `PostgresIndexer` as `StaticStringsIndexerDecorator ` is a not a good name for the service (and this is exposed via the config)

Reply:
ok, anyway this code is likely to change as soon as we plug another persistence.